apis/v1alpha1/httproute_types.go

```diff
@@ -174,16 +174,23 @@ type HTTPRouteRule struct {
 	// Conformance-levels at this level are defined based on the type of filter:
 	// - ALL core filters MUST be supported by all implementations.
 	// - Implementers are encouraged to support extended filters.
-	// - Implementation-specific custom filters have no API guarantees across implementations.
+	// - Implementation-specific custom filters have no API guarantees across
+	//   implementations.
+	//
 	// Specifying a core filter multiple times has unspecified or custom conformance.
 	//
+	// If unspecified, no filters will be applied to matched requests.
```

Comment:
I'm not sure this is necessary. It may also prevent future creative implementations where controllers might choose to apply a default set of filters for a given GatewayClass or something? No idea how useful that would be, but I think it would be safer to remove this instead of accidentally preventing that use case.